### PR TITLE
Simplify react markup of Home/Landing page

### PR DESCRIPTION
**Pull request by couds**
Simplify Home markup creating a new DestinationPicker component to encapsulate all the logic inside.

Also add Travelers picker

### CODE REVIEW

**Comment by cw00dw0rd** [comment]
@lostpebble could you review, please?

File: workspaces/frontend-react/src/screens/home/components/travelers-picker/travelers-picker.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import useonClickOutside from 'hooks/useOnClickOutside';
+
+import './travelers-picker.scss';
+
+const getTextValue = (value) => {
+  if (value.adults === 0) {
+    return '';
+  }
+  if (value.adults === 1 && value.children === 0) {
+    return 'and I travel alone';
```
Comment (lostpebble) [inline workspaces/frontend-react/src/screens/home/components/travelers-picker/travelers-picker.js:13]: "and I **will** travel alone" is more correct here.

File: workspaces/frontend-react/src/screens/home/components/travelers-picker/travelers-picker.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  }
+
+  return `and we are ${value.adults} adult${value.adults > 1 ? 's' : ''} ${value.children > 0 ? `and ${value.children} children` : ''}`.trim();
```
Comment (lostpebble) [inline workspaces/frontend-react/src/screens/home/components/travelers-picker/travelers-picker.js:16]: This is fine for now- but maybe in the future some icons and numbers would be better than a sentence? Its better to be more obvious about these selections for the user.

**Comment by lostpebble** [comment]
All looks good @couds - I also noticed that the second date input has a placeholder that says "to this dates" - just need to take the "s" off there.

Also, it seems I am able to select an earlier date than the "from date" in the "to date" input- some validation we should handle is to make that at least a day past the first date.

**Comment by cw00dw0rd** [comment]
Can this be merged?

**Comment by couds** [comment]
I need to fix some typos, I will do it today or tomorrow

**Comment by couds** [comment]
@cw00dw0rd you can merge it now =) Thanks

**Pull request merged**